docker-compose.yml

```diff
@@ -2,7 +2,8 @@ version: '2'
 
 services:
   web:
-    build: .
+#    build: .
+    image: business-logic
```

Comment:
Why is this added?

Reply:
It was just copied from the original business logic server template.  Reverted.